swagger.yml

```diff
@@ -800,15 +800,21 @@ definitions:
     required:
       - id
       - packageCount
+      - packageNumbers
       - status
       - timeline
     properties:
```

Comment:
**Nit:** Does it make sense to have response keys as "required"? 😂 When I think of "required", I think of what I need to include in the **_request_**, but not the response